src/default/assets/js/src/typedoc/services/Viewport.ts

```diff
@@ -83,7 +83,7 @@ namespace typedoc
          * Triggered when the user scrolled the viewport.
          */
         onScroll() {
-            this.scrollTop = $window.scrollTop() || 0;
+            this.scrollTop = window.scrollY || window.pageYOffset || 0;
```

Comment:
Other changes in this PR already drop support for IE 11, which I am fine with. We can drop the pageYOffset check here and just use scrollY.

Reply:
Fixed